src/scenes/hudScene.ts

```diff
@@ -190,6 +190,9 @@ export class HUDScene extends BaseScene {
     eventCenter.on(ClockEvents.CLOCK_MONTH_END, () => {
       progressMonth(this.GameState.Game, levels[this.GameState.Game.getCurrentLevel()]);
 
+      //cha ching
+      this.sound.play('cash', { volume: 0.5 });
```

Comment:
I think we should check if sound effects are enabled prior to playing, as otherwise this might not respect that settings. Could be worth putting that logic into its own method in case we add more sound effects in the future.